statsmodels/survey/survey_model.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+class SurveyModel(object):
+
+    def __init__(self, design, model_class, init_args={}, fit_args={}):
+        self.design = design
+        self.model = model_class
+        self.init_args = dict(init_args)
+        self.fit_args = dict(fit_args)
+
+    def _centering(self, array=None):
```

Comment:
Also we might want to move this _center function into SurveyDesign since it can be used by SurveyStat and other classes.